schemas/caching-schema.json

```
@@ -47,14 +47,64 @@
                                             "mode": {
                                                 "type": "string",
                                                 "description": "Type of storge in caching service.",
-                                                "enum": ["inMemory","VSAM","redis"],
+                                                "enum": ["inMemory","VSAM","redis","infinispan"],
                                                 "default": "inMemory"
                                             },
                                             "size": {
                                                 "type": "integer",
                                                 "description": "the number of records stored before the eviction strategy is initiated",
                                                 "default": 10000
                                             },
+                                            "infinispan":{
+                                                "type": "object",
+                                                "description": "When storage is selected infinispan",
```

Maybe something like this? Just a suggestion
```suggestion
                                                "description": "Use Infinispan as the selected storage solution",
```